### ISSUE TITLE
RGB difference

### ISSUE THREAD

**Issue by xyyu-kevin** [yjxiong/temporal-segment-networks#44]
Hello,  @yjxiong @wanglimin 

I noticed that you have experiment with adding stacked RGB difference as a input modality.

Can you tell me how to stacked RGB difference?
For new_length, modality, scale_ratios, what should I set?

Best regards

**Comment by 2502572025** [comment]
I have got RGB difference between consecutive frames. 
I training it with pre-train model, but accuracy is small than your accuracy (new_length = 1, modality = RGB, scale_ratios = 1,.875,.75,.66). 
And I training this model from scratch with new_length 10 (stacked 10 difference frames), the accuracy is worse(new_length = 10, modality = RGB, scale_ratios = 1,.875,.75,.66).
Whether new_length is not good or something wrong has happened?
When you stacked multi-frame as a input modality, how did you solve this matter without pre-train model? Just from scratch?
Best Regards

**Comment by JiqiangZhou** [comment]
How can I extract the RGB difference？

**Comment by jackyCold** [comment]
@2502572025 Can you tell me how to stacked RGB difference between consecutive frames?

**Comment by yjxiong** [comment]
For RGB diff you can refer to TSN-Pytorch.
